src/app/public/modules/select-field/select-field.component.spec.ts

```diff
@@ -188,11 +202,11 @@ describe('Select field component', () => {
 
       setValue([component.staticData[0]]);
       fixture.detectChanges();
-      expect(component.touched).toBe(1);
+      expect(component.touched).toBe(0);
 
       setValue([component.staticData[0]]);
       fixture.detectChanges();
-      expect(component.touched).toBe(1);
+      expect(component.touched).toBe(0);
```

Comment:
So, these values should have been 0 to begin with. The form should have never been `touched` when updating the form programmatically. I think this was just an oversight.

Reply:
Ah ignore my comment then! I’m not even sure it posted but that was the one thing I asked about